specials.lisp

```diff
@@ -78,6 +78,13 @@ variable has a true value, though.")
   "A `buffer' for one character.  Used by PEEK-CHAR* and
 UNREAD-CHAR*.")
 
+(defvar *max-line-length* nil
```

Comment:
Please add `declaim` of `type` for this and the other special var. I assume the correct type is `(or null unsigned-byte)`, but it may be `(or null (and fixnum unsigned-byte))` or something.